inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/enums/ReadPhase.java

```diff
@@ -0,0 +1,85 @@
+/*
+ *   Licensed to the Apache Software Foundation (ASF) under one
+ *   or more contributor license agreements.  See the NOTICE file
+ *   distributed with this work for additional information
+ *   regarding copyright ownership.  The ASF licenses this file
+ *   to you under the Apache License, Version 2.0 (the
+ *   "License"); you may not use this file except in compliance
+ *   with the License.  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.apache.inlong.sort.base.enums;
+
+import java.util.stream.Stream;
+
+/**
+ * A enum class for metrics read phase
+ */
+public enum ReadPhase {
+
+    /**
+     * This indicator represents the prepare phase during the read phase
```

Comment:
describe more details.What is prepare and snapshot and increase?

Reply:
> describe more details.What is prepare and snapshot and increase?

okay,I will add more description.